### PR TITLE
Bump changelog for 0.6.0

### PR DESCRIPTION
**Pull request by carlosms**
I've set the release date to next monday

---

* [x] I have updated the CHANGELOG file according to the conventions in [keepachangelog.com](https://keepachangelog.com)
* [ ] This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,18 +7,22 @@
...
+- Add new color palettes with source{d} brand colors ([#259](https://github.com/src-d/sourced-ui/issues/259)).
 
 ### Changed
 
-- Improve speed of development mode on macOs ([#266](https://github.com/src-d/sourced-ui/pull/266))
-- Make gunicorn to catch SIGTERM correctly which speed-ups stop command and makes container exit with correct exit code ([#239](https://github.com/src-d/sourced-ui/issues/239))
-- Optimize size of Docker image ([#275](https://github.com/src-d/sourced-ui/pull/275))
-
-</details>
+- Make source{d} brand the default color palette for charts ([#259](https://github.com/src-d/sourced-ui/pull/259)).
```
Comment (dpordomingo) [inline CHANGELOG.md]:
```suggestion
- Make source{d} brand the default color palette for charts ([#259](https://github.com/src-d/sourced-ui/pull/259)).
- Assign simpler palettes to some charts ([#258](https://github.com/src-d/sourced-ui/pull/258))
```

Comment (dpordomingo) [reply CHANGELOG.md]: :point_up: changelog suggestion for https://github.com/src-d/sourced-ui/pull/258

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,18 +7,22 @@
...
+- Improve speed of development mode on macOS ([#266](https://github.com/src-d/sourced-ui/pull/266)).
+- Make gunicorn catch SIGTERM correctly which speeds up the `docker stop` command, and makes the container exit with the correct exit code ([#239](https://github.com/src-d/sourced-ui/issues/239)).
+- Optimize size of the published Docker image, decreasing it from  1.76GB to 945MB ([#275](https://github.com/src-d/sourced-ui/pull/275)).
```
Comment (se7entyse7en) [inline CHANGELOG.md]:
```suggestion
- Optimize size of the published Docker image, decreasing it from 1.76GB to 945MB ([#275](https://github.com/src-d/sourced-ui/pull/275)).
```

**Comment by se7entyse7en** [comment]
Let's wait for #281 though.

**Comment by carlosms** [comment]
> Let's wait for #281 though.

Travis builds with fresh images, so it should not get into any bug with outdated images, right?

**Comment by se7entyse7en** [comment]
> Travis builds with fresh images, so it should not get into any bug with outdated images, right?

yup, travist not, but maybe some users could encounter your same issue with the non-fresh python image.

**Comment by dpordomingo** [comment]
[Travis is running](https://travis-ci.com/src-d/sourced-ui/builds/127769238) with suggested changes for #281.

**Comment by se7entyse7en** [comment]
@carlosms oh maybe I misinterpreted what you said 😂, yeah travis build wouldn't fail as the images would be new. My only concern was to release 0.6.0 after #281.

**Comment by dpordomingo** [comment]
#281 was merged :dancer:

**Pull request merged**